pages/secure/sec_adcs_certificates.rst

```diff
@@ -0,0 +1,285 @@
+      - CN=hostname8.mydomain.local
+      - dns:hostname8
+        dns:hostname8.mydomain.local
+        dns:graylogelasticalias.mydomain.local
+        dns:graylogelasticalias
+        ip:192.168.100.108
+      - PEM (certificate)
+        PKCS#8 (key file)
+    * - ElasticSearch 2
+      - CN=hostname9.mydomain.local
+      - dns:hostname9
+        dns:hostname9.mydomain.local
+        dns:graylogelasticalias.mydomain.local
+        dns:graylogelasticalias
+        ip:192.168.100.109
+      - PEM (certificate)
+        PKCS#8 (key file)
+    * - ElasticSearch 3
+      - CN=hostname9.mydomain.local
+      - dns:hostname9
+        dns:hostname9.mydomain.local
+        dns:graylogelasticalias.mydomain.local
+        dns:graylogelasticalias
+        ip:192.168.100.109
+      - PEM (certificate)
+        PKCS#8 (key file)
+
+
+Graylog stack certificate template
+===============
+
+The certificates for the Graylog stack and all of its components need some pretty specific settings. In order to achieve these, you will need to define a new certificate template in ADCS. 
+
+Be careful: 
+
+* Defining a new certicate template will require elevated privileges in your Active Directory domain. 
+* PKI and certificates are a matter of trust! Do not break your organization's *Certificate Policy* or its *Certificate practice Statement*. Stick to your standard procedures and do not simply start messing with the PKI!
+
+Defining the new template is done through the ADCS management tool "*Certification Authority*". 
+
+1. Duplicate the default ADCS WebServer template, rename it to your liking.
+2. General tab:
+  a. Set the name to something recognizable, for example "Graylog Stack Template".
+  b. The software will automatically generate the internal name, which removes all spaces: "GraylogStackTemplate".
+3. Cryptography tab:
+  a. **Provider** is the *Key Storage Provider*
+  b. **Requests can use any provider available on the subject's computer** is true
+  c. **Algorithm** is *RSA 2048*
+  d. **Request hash** is *SHA256*
+  e. **Use alternate signature hash** must be set to false.
+4. Extensions tab:
+  a. **Application policies** is set to both *server auth* as well as *client auth*.
+
+If you are going to be generating all the keypairs on your issuing CA or on another management station, then you will need to add the following as well, which will allow you to export the keypair for migration to the Graylog stack servers.
+
+5. Request handling tab:
+  a. Allow the private key to be exported is set to Yes.
+
+
+Generating the keypair and certificates - preparation
+===============
+
+The following instructions assume that you generate all the keypairs on a Windows administrative workstation, or on the issuing CA itself (meaning, you'll need that extra "*Allow the private key to be exported*" flag). You can of course generate all keys on the Graylog stack servers and then simply submit the CSR (certificate signing request) to the CA.
```

Comment:
Change "You can of course generate..." to "You can, of course, generate..."

Reply:
```suggestion
The following instructions assume that you generate all the keypairs on a Windows administrative workstation, or on the issuing CA itself (meaning, you'll need that extra "*Allow the private key to be exported*" flag). You can, of course, generate all keys on the Graylog stack servers and then simply submit the CSR (certificate signing request) to the CA.
```